.github/workflows/ci.yml

```diff
@@ -28,4 +28,4 @@ jobs:
         run: rustup show
       - uses: Swatinem/rust-cache@v2
       - name: Run tests
-        run: cargo test
+        run: pip install cairo-lang; cargo test
```

Comment:
Cam we setup python cleanly in the repo and just use the submodule cairo-lang instead of the pip one ?

Reply:
it's gonna be way slower so i'd rather not

Reply:
wdym? if we just checkout using the submodule no need to install anything even no?

Reply:
It’s not that easy check the docker file in cairo lang to see what are the steps to build the package from source